packages/synchro-charts/src/testing/app/routes.ts

```diff
@@ -9,10 +9,6 @@ export const routes = [
     url: '/',
     component: 'testing-ground',
   },
-  {
```

Comment:
Why was this removed?

Reply:
Because there is no tests int  the root path `/`